File: profiles/rne/saveicon.js
root.SaveIcon = {
    ForegroundAnimation: "SaveIcon",
    DefaultPosition: { X: 1153, Y: 23 },
    BackgroundSprite: "SaveIconBg",
    BackgroundOffset: { X: -7, Y: -4 },
    BackgroundMaxAlpha: 0.5
};

MakeAnimation({
    Name: "SaveIcon",
    Sheet: "Data",
    FirstFrameX: 1977,
    FirstFrameY: 1,
    FrameWidth: 70,
    ColWidth: 70,
    FrameHeight: 70,
    RowHeight: 72,
    Frames: 8,
    Duration: 0.5,
    Rows: 8,
    Columns: 1,
    PrimaryDirection: AnimationDirections.Down
});

root.Sprites["SaveIconBg"] = {
    Sheet: "Data",
    Bounds: { X: 1439, Y: 1, Width: 84, Height: 84 }
};

File: profiles/cc/hud/saveicon.lua
root.SaveIcon = {
    ForegroundAnimation = "SaveIcon",
    DefaultPosition = { X = 92, Y = 713 },
    BackgroundSprite = "SaveIconBg",
    BackgroundOffset = { X = 0, Y = 0 },
    BackgroundMaxAlpha = 0.0,
    FadeInDuration = 0.5,
    FadeOutDuration = 0.25
};

MakeAnimation({
    Name = "SaveIcon",
    Sheet = "Data",
    FirstFrameX = 115,
    FirstFrameY = 456,
    FrameWidth = 258,
    ColWidth = 258,
    FrameHeight = 178,
    RowHeight = 178,
    Frames = 2,
    Duration = 0.4,
    Rows = 2,
    Columns = 1,
    PrimaryDirection = AnimationDirections.Down
});

root.Sprites["SaveIconBg"] = {
    Sheet = "Data",
    Bounds = { X = 630, Y = 479, Width = 179, Height = 62 }
};